apps/web/src/components/StatusItem/StatusItem.tsx

```diff
@@ -0,0 +1,156 @@
+import * as React from "react";
+
+import CheckCircleIcon from "@heroicons/react/20/solid/CheckCircleIcon";
+import ExclamationCircleIcon from "@heroicons/react/20/solid/ExclamationCircleIcon";
+
+import { Link, IconType, ScrollToLink } from "@gc-digital-talent/ui";
+
+export type Status = "error" | "success";
+export type StatusColor = "default" | Status;
+
+const textColorMap: Record<StatusColor, Record<string, string>> = {
+  default: {
+    "data-h2-color": "base(black) base:hover(primary)",
+  },
+  error: {
+    "data-h2-color": "base(error.darker) base:hover(error.darkest)",
+  },
+  success: {
+    "data-h2-color": "base(success) base:hover(success.darker)",
+  },
+};
```

Comment:
I have no idea what the best way to resolve this is, but it does kind of bother me that the `span` also inherits the hover styles 😅 It isn't a big deal, just always tricks me into thinking I can click it. 

I'd hate to create an entirely different record for links at the same time though 😖

Reply:
I think that's intentional.  :shrug:  The hover styles were applied to non-links in the heros before I started this PR.

Reply:
I tried clicking the text once the hover animation happened😆

Reply:
I don't think this is show stopping enough to block on so I will resolve this and we can create a new ticket if we make a different decision that what is currently occurring. 👍